expts_manager/Expts_manager.yaml

```diff
@@ -0,0 +1,285 @@
+
+            PELAYOUT_attributes_combo:
+                ocn_ntasks: [168, 120]
+
+
+        config.yaml:
+            config_list1_combo:
+                ncpus: [192, 144]
+                mem: [768GB, 576GB]
+
+
+        ice_in:
+            shortwave_nml_combo:
+                albicei: [0.36, 0.39]
+                albicev: [0.78, 0.81]
+
+            ponds_nml_combo:
+                dpscale: [0.002, 0.003]
+
+
+        MOM_input:
+            MOM_list1_combo:
+                DT_THERM: [3600.0, 7200.0]
+                DIABATIC_FIRST: [False, False]
+                THERMO_SPANS_COUPLING: [True, True]
+                DTBT_RESET_PERIOD: [3600.0, 7200.0]
+
+
+        nuopc.runseq:
+            runseq_list1_combo:
+                cpl_dt: [1800.0, 7200.0]
+
+
+
+
+
+# ============ Perturbation Experiment Setup (Optional - access-om3) =======================
+# Configure settings for perturbation experiments. Currently, only `nuopc.runconfig` is supported.
+# If conducting parameter tuning for `nuopc.runconfig`, any pre-existing settings in this section 
+# will be purged by the above namelist tunning.
+
+perturb_run_config:
+    CLOCK_attributes:
+        stop_option: nyears
+        stop_n: 1
+        restart_option: nyears
+        restart_n: 1
+
+
+
+# ============ Control experiment and perturbation Runs ===================================
+# This section configures the settings for running control experiments and their corresponding perturbation tests.
+    
+ctrl_nruns: 0  
+# Number of control experiment runs.            
+# Default: 0.            
+# Adjust this value to set the number of iterations for the control experiment, which serves as the baseline for perturbations.
+
+run_namelists: True  
+# Determines whether to run using the specified namelists.            
+# Default: False.            
+# Set to 'True' to apply configurations from the namelist section; otherwise, 'False' to skip this step.
+
+check_duplicate_jobs: True  
+# Checks if there are duplicate PBS jobs within the same parent directory (`test_path`) based on their names.            
+# This check is useful when you have existing running jobs and want to add additional tests, which helps avoid conflicts by ensuring new jobs dont duplicate exisiting ones in the same `test_path`.            
+# The check will not be triggered if the jobs are located in different `test_path`. It only applies to jobs within the same `test_path` directory.            
+# Default: True.            
+# If duplicates are found, a message will be printed indicating the presence of duplicate runs and those runs will be skipped.
+
+check_skipping: False  
+# Checks if certain runs should be skipped based on pre-existing conditions or results. Currently only valids to nml type.            
+# Default: False.            
+# Set to 'True' if you want the system to skip runs under specific criteria; otherwise, keep it 'False'. Currently only valids to nml type.
```

Comment:
```suggestion
# Set to 'True' if you want the system to skip runs under specific criteria; otherwise, keep it 'False'. Currently only valid for nml type.
```